core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/LibraryTemplate.java

```diff
@@ -181,4 +185,21 @@ public void setIndexTwos(Map<String, Index> indexTwos) {
     this.indexTwos = indexTwos;
   }
 
+  @Override
+  public String getDeleteType() {
+    return "Library Template";
+  }
+
+  @Override
+  public String getDeleteDescription() {
+    // return getAlias() + " (" + getProjects().getAlias() + ")";
+    return getAlias();
+  }
+
+  @Override
+  public SecurityProfile getDeletionSecurityProfile() {
+    // return getProject().getSecurityProfile();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)